src/refract/JsonSchema.cc

```diff
@@ -138,9 +138,20 @@ namespace
         return schema;
     }
 
+    so::Object& addConst(so::Object& schema, so::Array value)
```

Comment:
Anhy particular reason to use `so::Array` for value instead of `so::Value`? Since it can only be one value, I don't see why we are wrapping that in an array.

Reply:
According how is `const` created - conversion form one member array i mean it is good way how to write it.

If you mean it is better just wrap one value I we can change 

line 151 to
```
addConst(schema, std::move(value.data.front());
```

And receive to addConst just one `so::Value`

Reply:
Maybe rename it `addConst()` to someting like `addEnumWithOneItem()`

Reply:
I think we should make the change so that the `addConst` takes a single value. That's how `const` is described in JSON Schema spec, it takes one value. We should leave the details of extracting the single value to the use of `addConst` (so in `addEnum`). A secondary benefit is that it simplifies the safety of `addConst` because `addConst` knows it only has a single value and we don't have to assert length of an array.